python/daqconf/core/conf_utils.py

```diff
@@ -361,7 +361,20 @@ def make_system_connections(the_system, verbose=False):
                     the_system.connections[publisher] += [pubsub_connectionids[connid]]
 
 
+def simplify_connections(app, connections):
```

Comment:
Hang on, why the need to "simplify" the connections? Why aren't they created with the right uri in the first place?